core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -449,8 +449,8 @@ object KafkaConfig {
       /** Internal Configurations **/
       // This indicates whether unreleased APIs should be advertised by this node.
       .defineInternal(ServerConfigs.UNSTABLE_API_VERSIONS_ENABLE_CONFIG, BOOLEAN, false, HIGH)
-      // This indicates whether unreleased MetadataVersions should be enabled on this node.
-      .defineInternal(ServerConfigs.UNSTABLE_METADATA_VERSIONS_ENABLE_CONFIG, BOOLEAN, false, HIGH)
+      // This indicates whether unreleased MetadataVersions or other feature versions should be enabled on this node.
+      .defineInternal(ServerConfigs.UNSTABLE_FEATURE_VERSIONS_ENABLE_CONFIG, BOOLEAN, false, HIGH)
```

Comment:
Do we want to keep it internal?

Reply:
Yeah, it's a fair question. I think the original intent was to keep it internal and only use for testing.

Reply:
Yup -- this is the text in the KIP:
> Add INTERNAL configuration unstable.feature.versions.enable to allow for non production ready features to be used (for testing)